{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/identifier"},"metadata":{"extendedModule":"VernissageServer","symbolKind":"property","role":"symbol","title":"identifier","roleHeading":"Instance Property","modules":[{"name":"VernissageServer"}],"externalID":"s:16VernissageServer11OpenAIErrorO10identifierSSvp","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}]},"sections":[],"primaryContentSections":[{"declarations":[{"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"identifier","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vernissageserver\/openaierror\/identifier"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/VernissageServer\/documentation\/VernissageServer","doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError"]]},"references":{"doc://VernissageServer/documentation/VernissageServer/OpenAIError/identifier":{"abstract":[],"type":"topic","role":"symbol","title":"identifier","url":"\/documentation\/vernissageserver\/openaierror\/identifier","identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/identifier","kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}]},"doc://VernissageServer/documentation/VernissageServer/OpenAIError":{"title":"OpenAIError","url":"\/documentation\/vernissageserver\/openaierror","role":"symbol","abstract":[{"type":"text","text":"Errors returned during using OpenAI endpoints."}],"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"OpenAIError"}],"navigatorTitle":[{"kind":"identifier","text":"OpenAIError"}],"kind":"symbol","identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError"},"doc://VernissageServer/documentation/VernissageServer":{"abstract":[{"text":"Application which is main API component for Vernissage photos sharing platform.","type":"text"}],"role":"collection","identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer","kind":"symbol","url":"\/documentation\/vernissageserver","title":"VernissageServer","type":"topic"}}}